concurrent/actor/ActorPeriodicTask.java

```diff
@@ -43,21 +39,14 @@ public ActorPeriodicTask(Actor<STATE> actor, int timeoutMs, int variationMs, Con
         timer = null;
     }
 
-    public void restart() {
```

Comment:
if you want a simple start, it to the reader it seems like it should have to call `stop` inside :)

Reply:
ganesh informs that there is no current requirement to do a simple "start" command, so you may not need it at all!